### ISSUE TITLE
Lobby: If Decider DCs while game is still in lobby, can reconnect to lobby w/o way to start game

### ISSUE THREAD

**Issue by mattwbenson** [engagementgamelab/at-stake-v3#268]
I had set up a game with five players with an iPad as the decider, and was still in the lobby before starting the game. I took a screencap on the iPad and uploaded it for a separate issue, and then I re-opened the native app on the iPad. When I did, the lobby screen was visible in the background, but I got the menu screen and error message requesting a refresh. The menu buttons were non-responsive (#254), but when I pressed on the red rectangle space where the error message displayed, the menu modal disappeared and the iPad went back to the name-entry screen. Because the other players hadn't been kicked from the lobby and the room code was still visible, I tried to re-enter the lobby with the iPad, as this seems a likely user scenario. When I did, the iPad reconnected as a normal player, and so the lobby had 5 players and no decider, and was therefore unable to ever actually start the game. 

This seems like more of a UX issue, where it needs to be clearer that if the decider disconnects before the game starts, that a new game lobby has to be created rather than using the same room code to rejoin. Otherwise, it can lead to a state where all players are in the lobby but nobody has a way to start the game, and error messaging doesn't appear or send players back to the start.
